Gems/ROS2/Code/Source/VehicleDynamics/Utilities.cpp

```diff
@@ -191,4 +191,24 @@ namespace ROS2::VehicleDynamics::Utilities
         return AZ::EntityComponentIdPair(wheelEntityId, hingeComponent->GetId());
     }
 
+    float ComputeRampVelocity(float targetVelocty, float lastVelocity, AZ::u64 deltaTimeNs, float acceleration, float maxVelocity)
+    {
+        const float deltaTimeSec = static_cast<float>(1e-9 *  static_cast<float>(deltaTimeNs));
+        const float eps = deltaTimeSec * acceleration; // minimal difference in speed to apply acceleration
+        float commandVelocity = 0;
+        if (AZStd::abs(lastVelocity - targetVelocty) < eps)
+        {
+            commandVelocity = targetVelocty;
+        }
+        else if (targetVelocty > lastVelocity)
+        {
+            commandVelocity = lastVelocity + deltaTimeSec * acceleration;
```

Comment:
`deltaTimeSec * acceleration` can be replaced by `eps` now.

So `eps` could be renamed to `deltaStep` or `deltaAcceleration` to be more clear.